core/src/rpc.rs

```diff
@@ -649,6 +665,34 @@ mod tests {
         assert_eq!(expected, result);
     }
 
+    #[test]
+    fn test_rpc_get_sol_total_supply() {
+        /* BOOTSTRAP_LEADER_LAMPORTS get minted for each
+         * bootstrap leader account:
+         * 1) Node account
+         * 2) Voting account
+         * 3) Staking account
+         *
+         * XXX: LEADER_ACCOUNT_COUNT needs to be updated if the number of accounts required
+         * for the bootstrap leader changes.
+         */
+        const LEADER_ACCOUNT_COUNT: u64 = 1 + 1 + 1;
```

Comment:
Got a little weird here. All ears as to a better solution!

Reply:
🙈 good enough